### ISSUE TITLE
Sample bug.

### ISSUE THREAD

**Issue by sola1tmy** [paulocns/kdb#3]
MVVM click can't goto next Activity.
SingleLiveEvent holds null. Null won't work for bind function.

````````
    fun <T> LiveData<T>.bind(function: (data: T) -> Unit) {
        observe(lifeCycleOwner, Observer { value ->
            value?.let {
                function(value)
            }
        })
    }
````````

Maybe T should be nullable?
I think Sometimes we can't make the value not null all the time.

**Comment by paulocns** [comment]
will be fixed on release 2.0.0 (next release)

**Comment by paulocns** [comment]
fixed on 2.0.0
